ydb/src/grpc_wrapper/raw_topic_service/common/codecs.rs

```diff
@@ -11,7 +11,7 @@ pub(crate) enum RawCodec {
     Custom(i32),
 }
 
-#[derive(serde::Serialize)]
+#[derive(serde::Serialize, Clone, Default)]
 pub(crate) struct RawSupportedCodecs {
     pub codecs: Vec<RawCodec>,
 }
```

Comment:
what about define fn RawCodec::from_int(codec: i32) and move map to the method?

```
                    RawCodec::Custom(val) => val,
                    RawCodec::Unspecified => Codec::Unspecified.into(),
                    RawCodec::Raw => Codec::Raw.into(),
                    RawCodec::Gzip => Codec::Gzip.into(),
                    RawCodec::Lzop => Codec::Lzop.into(),
                    RawCodec::Zstd => Codec::Zstd.into(),
```

it will allow deduplicate the map when convert from public Codec to RawCodec.